src/events/messageDelete.js

```diff
@@ -1,63 +1,65 @@
-const Discord = require('discord.js');
-const util = require('../util');
-
-/**
- * 
- * @param {Discord.Message} message
- */
-async function messageDeleteHandler(message) {
-	if (message.author.bot) return;
-	if (!message.member) return;
-
-	const guild = await message.guild.fetch();
-	const member = await message.member.fetch();
-	const user = member.user;
-
-	const messageContent = message.content.length > 1024 ? message.content.substr(0, 1023) + '…' : message.content;
-
-	const eventLogEmbed = new Discord.MessageEmbed();
-
-	eventLogEmbed.setColor(0xC0C0C0);
-	eventLogEmbed.setTitle('Event Type: _Message Delete_');
-	eventLogEmbed.setDescription('――――――――――――――――――――――――――――――――――');
-	eventLogEmbed.setTimestamp(Date.now());
-	eventLogEmbed.setFields(
-		{
-			name: 'User',
-			value: `<@${user.id}>`
-		},
-		{
-			name: 'User ID',
-			value: user.id
-		},
-		{
-			name: 'Author',
-			value: `<@${message.author.id}>`
-		},
-		{
-			name: 'Author ID',
-			value: message.author.id
-		},
-		{
-			name: 'Channel Tag',
-			value: `<#${message.channelId}>`
-		},
-		{
-			name: 'Channel Name',
-			value: message.channel.name
-		},
-		{
-			name: 'Message',
-			value: messageContent
-		}
-	);
-	eventLogEmbed.setFooter({
-		text: 'Pretendo Network',
-		iconURL: guild.iconURL()
-	});
-
-	await util.sendEventLogMessage(guild, message.channelId, eventLogEmbed);
-	
-}
-
+const Discord = require('discord.js');
+const util = require('../util');
+
+/**
+ * 
+ * @param {Discord.Message} message
+ */
+async function messageDeleteHandler(message) {
+	if (message.author.bot) return;
+
+	const guild = await message.guild.fetch();
+	const member = await message.member.fetch();
+	const user = member.user;
+
+	let messageContent = 'CONTENT UNAVAILABLE';
+
+	const eventLogEmbed = new Discord.MessageEmbed();
+	const image = new Discord.MessageAttachment('./src/images/events/event-delete.png');
+
+	eventLogEmbed.setAuthor({
+		name: user.tag,
+		iconURL: user.avatarURL()
+	});
+	eventLogEmbed.setColor(0xff6363);
+
+	eventLogEmbed.setTitle('_Poll Delete_');
+	eventLogEmbed.setDescription(`${user.username}'s poll in ${message.channel.name} has been deleted`);

```

Comment:
Why was this changed to only mention polls...? This should trigger on *any* message being deleted?

Reply:
It does, the function prior to this makes sure to only make these the string *if* the message was infact a poll. On any other occassion, it will show up as message deleted.

The image below shows off attachment deletion, poll deletion, and message deletion all working as intended.
![image](https://github.com/PretendoNetwork/chubby/assets/111249711/e1d77d93-38d2-4073-a2ac-9bf7d8f2bf55)

Reply:
I'm unsure how I feel about using just the message content check to assume it's a poll. Messages seem to have a `poll` field now, according to this screenshot from Discord? Does Discord.js support this already? Checking for the actual poll data feels like a much better check imo
![image (16)](https://github.com/PretendoNetwork/chubby/assets/27011796/4f91d3d2-09b5-4342-8467-4a68a2624848)

Reply:
No, Discord.js doesn’t support polls yet, which is why most of the poll deletion is hacky. Ideally, you would have some sort of “event/PollRemove.js” event which fires off when client detects a poll deletion.

*Technically*, you could call some function after poll deletion in “event/MessageCreate.js”, but again not ideal.